nixosConfigurations/hetzner-ephemeral-alpha/default.nix

```diff
@@ -126,134 +111,13 @@ in {
       Type = "simple";
     };
 
-    script = ''      ${pkgs.blutgang}/bin/blutgang \
+    script = ''      /var/mnt/blutgang-0.3.0/bin/blutgang \
```

Comment:
Care to explain this?

Reply:
Blutgang has a bug in which the first message from a Websocket subscribe command returns malformed JSON, which crashes keep-client.  So, I built Blutgang unstable from source. I copied the binary to the server while I'm waiting for a new upstream release, which I will then submit to ethereum.nix